app/views/members/_search.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_tag(members_path, method: :get) do %>
+  <%= text_field_tag :query, params[:query] %>
+  <%= submit_tag 'Search', name: nil %>
```

Comment:
Maybe it's not important for the URL?  That is a big question I have generally.  How 'clean' should a URL be?